test/test_arrays.py

```diff
@@ -1,144 +0,0 @@
-import bson
```

Comment:
I've reviewed the tests here and either moved them or decided they were redundant.